content/_help/verify-your-identity/phone-number._es.md

```diff
@@ -20,15 +20,18 @@ Si ingresa su número principal (el que usa más a menudo), es más probable que
 Puede utilizar un teléfono celular de prepago para verificar su identidad. Sin embargo, algunos números de teléfono de prepago tienen factores de riesgo que pueden impedir la verificación de su identidad.
 
 ## No puede verificar su identidad con:
-- Números internacionales
+- Números internacionales y números en algunos territorios de EE.UU.
 - Google Voice o números VoIP (*Voice Over Internet Protocol*) similares
```

Comment:
The italics make me think leaving this in English was intentional? But VoIP is capitalized correctly.

Reply:
Let's lowercase the "o" in "Over" here too.